doc/rst/source/datasets/remote-data.rst

```diff
@@ -41,6 +41,12 @@ do specify a specific registration and that version is not available you will ge
 The codes for *rr*\ *u* and the optional *reg* that are supported will be listed in the sections
 below describing each of the available data sets.
 
+When used in plots (i.e., both when a region and map projection is selected to make an image) the data
+resolution is optional. If it is not given then we determine a data set resolution that will result
+in a final plot image dots-per-unit resolution that is the closest to the :term:`GMT_GRAPHICS_DPU` default
+setting. This eliminates the need for the user to determine what grid resolution will give a nice-looking
+image and not create a bloated file that exceeds what the eye (or printers) can discern. **Note**: Grid
+processing tools require the data resolution to be specified since no plot is being generated.
```

Comment:
```suggestion
image and not create a bloated file that exceeds what the eye (or printers) can discern. Use
:doc:`/grdcut` with the **-D** option to inquire about the automatic resolution. **Note**: Grid
processing tools require the data resolution to be specified since no plot is being generated.
```